invideoquiz/public/js/src/config.js

```diff
@@ -6,7 +6,12 @@ var InVideoQuizXBlock = InVideoQuizXBlock || {{}};
     InVideoQuizXBlock.config = InVideoQuizXBlock.config || {{}};
 
     var videoId = '{video_id}';
-    if (videoId) {{
-        InVideoQuizXBlock.config[videoId] = {timemap};
+    try {{        
+        if (videoId) {{
+            InVideoQuizXBlock.config[videoId] = JSON.parse(`{timemap}`.replace(/\'/g,`"`));
```

Comment:
In JSON, only double quote is recognized, so I replace single quote with double quote.

Reply:
Along with my comment below, we should check for this during/prior to saving the value and raise a validation error onSave, instead of allowing an invalid value to be saved and then corrected upon each load/runtime.

Doing so, as prescribed below, should also remove the need for this change here.

Reply:
@iamCristYe I think we can drop this change/block here, now that validation is done on the backend, right?

Reply:
I've deleted the conversion for `'` to `"`, as they are now verified backend in Python